saleor/dashboard/staff/views.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import unicode_literals
+
+from django.shortcuts import get_object_or_404
+from django.template.response import TemplateResponse
+
+from saleor.core.permissions import get_user_permissions, update_permissions
```

Comment:
We use relative imports when importing local modules.

Reply:
A ok, this was autogenerated when i refactor in PyCharm